src/main/java/org/kainos/ea/data/UserData.java

```diff
@@ -65,4 +65,37 @@ public boolean removeToken( Connection conn, String email, String token ) throws
         return false;
 
     }
+
+    public String generateHash( String password ) {
```

Comment:
this should be a private method so it doesn't require testing

Reply:
changed to private